packages/go_router/lib/src/router.dart

```diff
@@ -496,12 +496,17 @@ class GoRouter implements RouterConfig<RouteMatchList> {
   ///
   /// If the top-most route is a pop up or dialog, this method pops it instead
   /// of any GoRoute under it.
+  ///
+  /// @restore()
```

Comment:
these are implementation detail, I would remove this from the doc to avoid confusing the reader